static/app/views/performance/newTraceDetails/traceDrawer/details/transaction/index.tsx

```diff
@@ -102,7 +107,17 @@ export function TransactionNodeDetails({
     organization,
   });
 
-  if (isLoading) {
+  const {data: cacheMetrics, isLoading: isCacheMetricsLoading} = useSpanMetrics(
+    {
+      search: MutableSearch.fromQueryObject({
+        transaction: node.value.transaction,
+      } satisfies SpanMetricsQueryFilters),
+      fields: ['avg(cache.item_size)', 'cache_miss_rate()'],
+    },
+    Referrer.TRACE_DRAWER_TRANSACTION_CACHE_METRICS
+  );
+
+  if (isLoading || isCacheMetricsLoading) {
```

Comment:
Can this be independent from the first request? I'm not a fan of showing loading indicators while multiple requests load, we should instead aim to flush things to the screen asap

Reply:
We could, but I wanted to avoid the case where the cacheMetrics take longer to load, and suddenly pop into view causing a layout shift. If we're okay with that shift then I can make this independent. 

Alternatively we could do some sort of skeleton for this data, but cacheMetrics won't always be applicable to a transaction, so in that case we probably shouldn't show them.

Reply:
I think it's ok to show it into the screen later, layout shift is a better problem to have than slow load times. A skeleton sounds like a good idea, and can always be added later. Would you mind sending me a screencap of how this loads when we do have cached metrics vs when we dont? I'm going to show them to the designer on this so we can figure out how what kind of experience we can show to progressively enhance or degrade this

Reply:
Sent over slack!